<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-GEL415J" name="bg.codix.chat.repository.UserRepositoryTest" tests="10" failures="0" timestamp="2021-10-31T16:16:23 EET" time="0.012" errors="0">
  <testcase name="getUserByUsernameCorrect" time="0.001" classname="bg.codix.chat.repository.UserRepositoryTest"/>
  <system-out/>
  <testcase name="getUserByIdCorrect" time="0.001" classname="bg.codix.chat.repository.UserRepositoryTest"/>
  <system-out/>
  <testcase name="getUserByUsernameHandleException" time="0.000" classname="bg.codix.chat.repository.UserRepositoryTest"/>
  <system-out/>
  <testcase name="checkSameGroupOfUsersCorrect" time="0.002" classname="bg.codix.chat.repository.UserRepositoryTest"/>
  <system-out/>
  <testcase name="getAllByGroup" time="0.002" classname="bg.codix.chat.repository.UserRepositoryTest"/>
  <system-out/>
  <testcase name="getUsersWhoCanSendMessageToCorrect" time="0.002" classname="bg.codix.chat.repository.UserRepositoryTest"/>
  <system-out/>
  <testcase name="getAllByGroupHandleException" time="0.001" classname="bg.codix.chat.repository.UserRepositoryTest"/>
  <system-out/>
  <testcase name="getUsersWhoCanSendMessageToHandleException" time="0.000" classname="bg.codix.chat.repository.UserRepositoryTest"/>
  <system-out/>
  <testcase name="checkSameGroupOfUsersHandleException" time="0.002" classname="bg.codix.chat.repository.UserRepositoryTest"/>
  <system-out/>
  <testcase name="getUserByIdCorrectHandleException" time="0.001" classname="bg.codix.chat.repository.UserRepositoryTest"/>
  <system-out/>
</testsuite> <!-- bg.codix.chat.repository.UserRepositoryTest -->
